waku/message_test.go

```diff
@@ -26,9 +26,11 @@ import (
 	"testing"
 	"time"
 
```

Comment:
@cammellos the linter is complaining that this line is failing on file formatting ground https://ci.status.im/blue/organizations/jenkins/status-go%2Fprs%2Ftests/detail/PR-1950/1/pipeline . I don't see anything at all wrong with it. 

Are you able to see any thing? Does your `gofmt` give anything different?

Reply:
resolved with https://github.com/status-im/status-go/pull/1950/commits/04413b9e31e79230f55531cd34cd1481225da993